azurecompute-arm/src/main/java/org/jclouds/azurecomputearm/features/StorageAccountApi.java

```diff
@@ -16,125 +16,146 @@
  */
 package org.jclouds.azurecomputearm.features;
 
-import java.util.List;
-
 import javax.inject.Named;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.PUT;
+import javax.ws.rs.POST;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.core.MediaType;
 
 import org.jclouds.Fallbacks;
 import org.jclouds.Fallbacks.NullOnNotFoundOr404;
-import org.jclouds.azurecomputearm.binders.StorageServiceKeyTypeToXML;
-import org.jclouds.azurecomputearm.binders.CreateStorageServiceParamsToXML;
-import org.jclouds.azurecomputearm.binders.UpdateStorageServiceParamsToXML;
 import org.jclouds.azurecomputearm.domain.Availability;
+import org.jclouds.azurecomputearm.domain.CreateStorageServiceParams;
 import org.jclouds.azurecomputearm.domain.StorageService;
 import org.jclouds.azurecomputearm.domain.StorageServiceKeys;
-import org.jclouds.azurecomputearm.domain.StorageServiceKeys.KeyType;
-import org.jclouds.azurecomputearm.domain.CreateStorageServiceParams;
-import org.jclouds.azurecomputearm.domain.UpdateStorageServiceParams;
-import org.jclouds.azurecomputearm.functions.ParseRequestIdHeader;
-import org.jclouds.azurecomputearm.xml.AvailabilityHandler;
-import org.jclouds.azurecomputearm.xml.ListStorageServicesHandler;
-import org.jclouds.azurecomputearm.xml.StorageServiceHandler;
-import org.jclouds.azurecomputearm.xml.StorageServiceKeysHandler;
-import org.jclouds.rest.annotations.BinderParam;
+import org.jclouds.azurecomputearm.domain.StorageServiceUpdateParams;
+import org.jclouds.azurecomputearm.functions.StatusCodeParser;
+import org.jclouds.azurecomputearm.oauth.v2.filters.OAuthFilter;
 import org.jclouds.rest.annotations.Fallback;
 import org.jclouds.rest.annotations.Headers;
 import org.jclouds.rest.annotations.QueryParams;
+import org.jclouds.rest.annotations.RequestFilters;
+import org.jclouds.rest.annotations.SelectJson;
+import org.jclouds.rest.annotations.Payload;
+import org.jclouds.rest.annotations.PATCH;
 import org.jclouds.rest.annotations.ResponseParser;
-import org.jclouds.rest.annotations.XMLResponseParser;
+import org.jclouds.rest.annotations.PayloadParam;
+import org.jclouds.rest.annotations.MapBinder;
+import org.jclouds.rest.binders.BindToJsonPayload;
+
+import java.util.List;
+import java.util.Map;
 
 /**
  * The Service Management API includes operations for managing the storage accounts in your subscription.
  *
  * @see <a href="http://msdn.microsoft.com/en-us/library/azure/ee460790">docs</a>
  */
-@Path("/services/storageservices")
 @Headers(keys = "x-ms-version", values = "{jclouds.api-version}")
```

Comment:
This is also changed now. I propose that for rest of these we create own ticket where we create one cleaning commit where we change all of these and also remove some obsolete files etc. I go and create ticket to trello